provisioner/group_vars/all/vpc_rules.yml

```diff
@@ -60,6 +60,26 @@ workshops:
         from_port: 514
         cidr_ip: 0.0.0.0/0
         rule_desc: WinRM
+      - proto: tcp
+        to_port: 5432
```

Comment:
Same here, I would make Tower use the internal IP to access the DB versus the Public IP.  That way we don't have to open Postgres to the world.

Reply:
Updated to use internal IPs